pkg/cluster/bootstrap.go

```diff
@@ -567,12 +568,19 @@ func (c *Cluster) ReconcileBootstrapData(ctx context.Context, buf io.ReadSeeker,
 				updateDisk = true
 				continue
 			}
-			logrus.Fatal(path + " newer than datastore and could cause cluster outage. Remove the file from disk and restart to be recreated from datastore.")
+			newerOnDisk = append(newerOnDisk, path)
 		case res.conflict:
 			logrus.Warnf("datastore / disk conflict: %s newer than in the datastore", path)
 		}
 	}
 
+	if len(newerOnDisk) > 0 {
+		if len(newerOnDisk) == 1 {
+			logrus.Fatal(newerOnDisk[0] + " newer than datastore and could cause a cluster outage. Remove the file from disk and restart to be recreated from datastore.")
+		}
+		logrus.Fatal(strings.Join(newerOnDisk, ", ") + " newer than datastore and could cause a cluster outage. Remove the files from disk and restart to be recreated from datastore.")
```

Comment:
Aren't these effectively the same? Joining a single-element list should give you the same result as just printing the only element.

Reply:
No comma and different verbiage between them.

Reply:
You don't get a trailing comma if there's only one element, and I think we could handle the potential pluralization without having to have the whole text of the message repeated?

https://go.dev/play/p/fVDkQdeqGxM